RevenueCatUI/CustomerCenter/Views/CustomerCenterView.swift

```diff
@@ -43,29 +43,53 @@ public struct CustomerCenterView: View {
 
     private let mode: CustomerCenterPresentationMode
 
+    /// A flag indicating whether the view is already embedded in a navigation stack.
+    ///
+    /// - When set to `true`, the view must be part of an existing NavigationStack / NavigationView.
```

Comment:
Small detail. I'd give NavigationStack and NavigationView code formatting:
* NavigationStack --> `NavigationStack`
* NavigationView --> `NavigationView`